src/main/java/org/myrobotlab/service/LocalSpeech.java

```diff
@@ -281,6 +284,8 @@ public void removeExt(boolean b) {
   public boolean setEspeak() {
     LocalSpeechConfig c = (LocalSpeechConfig)config;
     c.speechType = "Espeak";
+    voices.clear();
+    addVoice("Linus", "male", "en-US", "festival");

```

Comment:
Not sure this is meant to be in setEspeak()

Reply:
the "festival" is a copy paste error, but I did want to put a voice in each voice subtype, because our voice definition should be the way to further distinguish other voices or language support in the subtypes ... I did this to pico2wav so thought I would add at least 1 voice to each. Correcting the Espeak festival part though.  Hmm, i guess since we only have one (at the moment) we'll call it espeak voice .. :shrug: 
fixed